diff --git a/install.sh b/install.sh
index 5434618..17414bb 100755
--- a/install.sh
+++ b/install.sh
@@ -1,9 +1,9 @@
 cd ./src/esprima/ && npm i && cd -;
 npm install \
-    @babel/core \
-    @babel/cli \
-    @babel/preset-env \
-    @babel/preset-typescript \
+    @babel/core@7.15.5 \
+    @babel/cli@7.15.7 \
+    @babel/preset-env@7.15.6 \
+    @babel/preset-typescript@7.15.0 \
     mocha;
 python3 -m pip install -r ./requirements.txt;
 
